litmus-portal/frontend/src/components/Sections/Settings/UserManagementTab/CreateUser/UserDetails/index.tsx

```diff
@@ -53,45 +58,31 @@ const UserDetails: React.FC<PersonalDetailsProps> = ({
           </div>
           {/* Fields for details including Full name, email, username */}
           <div className={classes.details1}>
-            <TextField
+            <InputField
```

Comment:
Add a helper text here too based on the validation, take reference from `Stepper.tsx`